packages/isolate_bloc/example/lib/main.dart

```diff
@@ -34,13 +34,13 @@ class CounterScreen extends StatelessWidget {
         children: [
           FloatingActionButton(
             heroTag: 'Increment',
-            onPressed: () => context.isolateBloc<CounterBloc, int>().add(CountEvent.increment),
+            onPressed: () => context.isolateBloc<CounterCubit, int>().add(CountEvent.increment),
```

Comment:
Why is it a Cubit? The API looks like BLoC.

Reply:
IsolateBloc and IsolateCubit's apis looks really the same but they have different implementations

Quote from new api discussion
> The main idea of onEventReceived method is its simlicity. You must not deal with complex async generators or even more complex situations when you need to update state without ui intent. You just receive event and emit state at any place
> Maybe we should just add mapEventToState to the existing IsolateBloc or separated entities for each approach? Bloc with onEventReceived may be some kind of IsolateCubit while bloc with mapEventToState may be IsolateBloc